jaeger-core/src/main/java/com/uber/jaeger/SpanContext.java

```diff
@@ -177,4 +177,44 @@ public static SpanContext withDebugId(String debugId) {
   String getDebugId() {
     return debugId;
   }
+
+  /**
+   * @deprecated use {@link SpanContext#getTraceId()} instead.
+   */
+  @Deprecated
+  public long getTraceID() {
+    return traceId;
+  }
+
+  /**
+   * @deprecated use {@link SpanContext#getSpanID()} instead.
+   */
+  @Deprecated
+  public long getSpanID() {
+    return spanId;
+  }
+
+  /**
+   * @deprecated use {@link SpanContext#getParentID()} instead.
+   */
+  @Deprecated
+  public long getParentID() {
+    return parentId;
+  }
+
+  /**
+   * @deprecated use {@link SpanContext#getDebugId()} instead.
+   */
+  @Deprecated
```

Comment:
these last two are not needed, external dependencies are only on the trace/span id related fields